angular-holiday-planner/src/app/components/trip-tile/trip-tile.component.html

```diff
@@ -0,0 +1,21 @@
+<!--Use evaluated class attribute for the row numbers in the template-->
+<!--And once you set up the backend, when running an ngFor on this-->
+<!--Use the length of the array + 1 for the headings for the rows and index to specify-->
+<!--Which row the created container is going to be on-->
+
+<div class="events-section ">
+  <div class=" grid grid-cols-[repeat(3, 1fr)] grid-rows-[repeat(1, 3rem)] p-5 bg-slate-500 rounded-t-xl">
+    <h1 class="col-[1/2] row-[1/2] text-center">Name</h1>
+    <h1 class="col-[2/3] row-[1/2] text-center">Location</h1>
+  <!--  <h1 class="col-[3/4] row-[1/2]">Itinerary Count</h1>-->
+    <h1 class="col-[3/4] row-[1/2] text-center">Cost</h1>
```

Comment:
A page should only have 1 h1 I would say rather use p or span for each and set the text size or wrap it all in a h1, I think that should be good

Reply:
Will do, thanks 👍🏽